### DISCUSSION TITLE
Laravel 11 with teams: how to add middleware

### DISCUSSION THREAD

**Discussion by giacomomasseron** [spatie/laravel-permission#2728] [Q&A]
Hi,  
in Laravel 11 the default middlewares have been removed from the code.

I'm trying to use teams and to add the middleware written in the documentation (https://spatie.be/docs/laravel-permission/v6/basic-usage/teams-permissions)

I'm not able retrieve the logged user in the middleware.

This is what I tried without success (`laravel bootstrap/app.php` file):

	return Application::configure(basePath: dirname(__DIR__))
		->withRouting(
			web: __DIR__.'/../routes/web.php',
			commands: __DIR__.'/../routes/console.php',
			health: '/up',
		)
		->withMiddleware(function (Middleware $middleware) {
			$middleware->append([
				\App\Http\Middleware\TeamsPermission::class,
			]);
		})
		->withExceptions(function (Exceptions $exceptions) {
			//
		})->create();

With this code, in the middleware, `auth()->user()` is null even if I'm authenticated

**Comment by drbyte** [comment]
Is `$request->user()` also null?

**Reply by giacomomasseron** [reply]
Yes, it is null
